File: src/com/binarysushi/studio/toolWindow/StudioConsoleService.java
package com.binarysushi.studio.toolWindow;

import com.intellij.execution.filters.TextConsoleBuilderFactory;
import com.intellij.execution.ui.ConsoleView;
import com.intellij.openapi.project.Project;

public class StudioConsoleService {
    private final ConsoleView consoleView;

    public StudioConsoleService(Project project) {
        consoleView = TextConsoleBuilderFactory.getInstance().createBuilder(project).getConsole();
    }

    public ConsoleView getConsoleView() {
        return consoleView;
    }
}

File: src/com/binarysushi/studio/toolWindow/StudioConsoleService.kt
package com.binarysushi.studio.toolWindow

import com.intellij.execution.filters.TextConsoleBuilderFactory
import com.intellij.execution.ui.ConsoleView
import com.intellij.openapi.project.Project

class StudioConsoleService(project: Project) {
    val consoleView = TextConsoleBuilderFactory.getInstance().createBuilder(project).console
}
